### PR TITLE
docs: Missing plugins doc of apisix shown in website

### PR DESCRIPTION
**Pull request by dhawal-kamdar**
### What this PR does / why we need it:

There are some missing docs plugins of apisix on the website.

We could find the missing plugins in the doc folder of the apisix repo, but they are not present on the website, because they are not added in the "config.json" file.
So this PR fixes the above issue, as all the missing plugins are added in the "docs/en/latest/config.json" and "docs/zh/latest/config.json" file.

Fix: https://github.com/apache/apisix-website/issues/236

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — CHANGES_REQUESTED** [review]
Should also update the docs/zh/latest/config.json

**Comment by dhawal-kamdar** [comment]
> Should also update the docs/zh/latest/config.json

Okay 👍🏻

File: docs/zh/latest/config.json [hunk 1]
```diff
@@ -86,7 +92,17 @@
...
             "plugins/syslog",
-            "plugins/log-rotate"
+            "plugins/log-rotate",
+            "plugins/error-log-logger",
+            "plugins/sls-logger"
+          ]
+        },
+        {
+          "type": "category",
+          "label": "Other Protocols",
```
Comment (juzhiyuan) [inline docs/zh/latest/config.json:102]:
```suggestion
          "label": "其它",
```

**Review by juzhiyuan — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/config.json (hunk 1 above)
Comment (dhawal-kamdar) [reply docs/zh/latest/config.json:102]: @juzhiyuan should I commit these changes?

Comment (juzhiyuan) [reply docs/zh/latest/config.json:102]: Let me help you

**Comment by dhawal-kamdar** [comment]
Hello, @juzhiyuan can you please help me figure out why one check is failing?
In details, it is showing this error:

image

**Comment by juzhiyuan** [comment]
@dhawal-kamdar Hi, let's wait for others help :)

**Comment by dhawal-kamdar** [comment]
> @dhawal-kamdar Hi, let's wait for others help :)

Okay👍🏻

**Comment by spacewander** [comment]
> 
> 
> Hello, @juzhiyuan can you please help me figure out why one check is failing?
> In details, it is showing this error:
> 
> image

The check is passed. The error message is expected.

**Comment by spacewander** [comment]
> 
> 
> Hello, @juzhiyuan can you please help me figure out why one check is failing?
> In details, it is showing this error:
> 
> image

The test failed because:
```
#   Failed test 'TEST 1: generate different random number in different worker process - response_body - response is expected (repeated req 0, req 0)'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1589.
# @@ -1,4 +1,4 @@
#  random[1] == random[2]: false
#  random[2] == random[3]: false
#  random[3] == random[4]: false
# -random[4] == random[5]: false
# +random[4] == random[5]: true
# Looks like you failed 1 test of 3.
t/core/random.t ............................ 
Dubious, test returned 1 (wstat 256, 0x100)
```

Look like this case is unstable.

The error message in the image is expected.

**Pull request merged**